### PR TITLE
make validate function have return

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/kamontia/qs/issues/115

### CODE REVIEW

**Comment by chaspy** [comment]
@kamontia Please review and merge

File: main.go [hunk 1]
```diff
@@ -250,7 +246,11 @@ func main() {
 		/* Create GitCommit Info */
 		gci := new(model.GitCommitInfo)
 
-		validate(c.String("number"))
+		if !validate(c.String("number")) {
```
Comment (kamontia) [inline main.go]: MatchString returns `bool` type and `error` type.
In golang, I suppose that it is preferable to implement following.
```
matched , error := validate(c.String("number"))

if matched != false or error != nil {
  ...
}
```

Comment (kamontia) [reply main.go]: ref: https://golang.org/pkg/regexp/#MatchString

Comment (kamontia) [reply main.go]: @chaspy  Uoops, I'm sorry to misunderstand spec of `MatchString` function.

`MatchSring` function has overload.

1. `func MatchString(pattern string, s string) (matched bool, err error)`
1. `func (re *Regexp) MatchString(s string) bool`

You choiced second pattern, so could you revert?

**Comment by chaspy** [comment]
@kamontia 
Please review and merge

**Comment by kamontia** [comment]
@chaspy OK. No problem. Thank you for your implementation.

**Pull request merged**